### ISSUE TITLE
Failed to compile on MacOSX

### ISSUE THREAD

**Issue by mtangoo** [iwbnwif/wxFreeChart#34]
`./configure --with-wxdir=../wxWidgets/` Runs fine
`make`

```
/Users/libs/wxFreeChart/bk-deps g++ -c -o wxfreechart_dll_numberaxis.o -I./include -DWXMAKINGDLL_FREECHART -dynamic -fPIC -DPIC -I/Users/libs/wxWidgets/lib/wx/include/osx_cocoa-unicode-3.1 -I/Users/hosanna/Projects/libs/wxWidgets/include -D_FILE_OFFSET_BITS=64 -DWXUSINGDLL -D__WXMAC__ -D__WXOSX__ -D__WXOSX_COCOA__   -O2 -fno-common ./src/axis/numberaxis.cpp
./src/axis/numberaxis.cpp:34:18: error: expected unqualified-id
    switch (std::fpclassify(v)) {
                 ^
/usr/include/math.h:140:5: note: expanded from macro 'fpclassify'
    ( sizeof(x) == sizeof(float)  ? __fpclassifyf((float)(x))            \
    ^
1 error generated.
make: *** [wxfreechart_dll_numberaxis.o] Error 1
```

Do I miss anything here?

**Comment by iwbnwif** [comment]
Please can you try removing the std:: to see if that helps on OSX. It should then use the C99 macro rather than the STL version.

**Comment by mtangoo** [comment]
ok, i will try

**Comment by mtangoo** [comment]
It compiled fine when removing std

**Comment by iwbnwif** [comment]
Fixed in 67760cd
